### PR TITLE
Sensors

### PR DESCRIPTION
**Pull request by fielia**

### CODE REVIEW

**Comment by vladaviedov** [comment]
Wrong destination branch!

**Comment by fielia** [comment]
my bad... at least it's fixed now

File: src/main/java/org/recordrobotics/munchkin/RobotMap.java [hunk 1]
```diff
@@ -66,4 +63,18 @@ public class Rotator {
 		public static final int BWD_LIMIT_PORT = 1;
 
 	}
+
+	/**
+	 * Miscellaneous Sensor Ports
+	 */
+	public class Sensors {
+		// TODO: set port type
```
Comment (vladaviedov) [inline src/main/java/org/recordrobotics/munchkin/RobotMap.java:71]: These are DIO ports

**Review by vladaviedov — COMMENTED** [review]
Looks good, change the comment and I'll approve

**Review by vladaviedov — APPROVED** [review]

**Pull request merged**